OrbitCore/OrbitSession.h

```diff
@@ -2,20 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#pragma once
+#ifndef ORBIT_CORE_ORBIT_SESSION_H_
```

Comment:
While you're here, this file should probably be renamed to `OrbitPreset.h` (or just `Preset.h` I guess).

Reply:
I probably want to replace this struct with proto and remove the file in the next PR

Reply:
OrbitPreset.h was removed in #901